notebook/agentchat_sql.ipynb

```diff
@@ -0,0 +1,858 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6f384223",
+   "metadata": {},
+   "source": [
+    "# SQL Agent\n",
+    "\n",
+    "\n",
+    "Highlight:\n",
+    "1. If the schema of the database is too big to fit the content length, such as too many tables or too many columns, the Agent will truncate the schema and will ask the LLM to select tables on the fly.\n",
+    "    a. We still recommend increasing the content size as much as possible, as more information can be significantly beneficial.\n",
```

Comment:
This part is not preperly rendered, better replace `a.` with `-`:

![image](https://github.com/microsoft/autogen/assets/7947027/90f8ec99-8481-4996-9c77-67480ded61b6)